LEAF_Request_Portal/templates/view_status.tpl

```diff
@@ -28,7 +28,11 @@ Title of request: <a href="?a=printview&amp;recordID=<!--{$recordID|strip_tags|e
     <td>
         <span><b><!--{$indicator.description|sanitize}--></b> by <!--{$indicator.userName|sanitize}-->
         <!--{if $indicator.comment != ''}-->
-        <br />Comment: <!--{$indicator.comment|sanitize}--></span>
+            <!--{if $indicator.description != 'Email Sent: '}-->
```

Comment:
I don't recall if there is an option for piping to lower case for this comparison in smarty.

Reply:
added.